examples/ex4.py

```diff
@@ -0,0 +1,32 @@
+# List of ipymd cells expected for this example.
+output = [
+
+    {'is_notebook': True,
+     'metadata': {'title': 'A Slideshow'}},
```

Comment:
Having the cell metadata as a particular ipymd cell with `is_notebook=True` is not the most elegant thing in the world, but I guess that was the simplest option that didn't require major changes in the code? Could we have a slightly more general mechanism, like:

```
{
'cell_type': 'notebook_metadata',
'metadata': {...}
}
```
? This way we could reuse the existing `cell_type` property for that purpose, instead of introducing a new one.